packages/app/cypress/e2e/reporter_header.cy.ts

```diff
@@ -0,0 +1,84 @@
+describe('Reporter Header', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer()
+    cy.visitApp()
+    cy.contains('dom-content.spec').click()
+  })
+
+  context('Specs Shortcut', () => {
+    it('displays the currently running spec as highlighted in the Specs List', () => {
+      cy.location().should((location) => {
+        expect(location.hash).to.contain('dom-content.spec')
+      })
+
+      cy.get('[data-selected-spec="true"').should('contain', 'dom-content')
+      cy.get('[data-selected-spec="false"').should('have.length', '1')
+    })
+
+    it('will display the extension in the list as a different highlighted color than the filename', () => {
+      cy.location().should((location) => {
+        expect(location.hash).to.contain('dom-content.spec')
+      })
+
+      cy.get('[data-testid=spec-file-item]').get('.text-gray-700')
+      .should('contain', '.')
+      .should('contain', 's')
+      .should('contain', 'p')
+      .should('contain', 'e')
+      .should('contain', 'c')
+      .should('contain', '.')
+      .should('contain', 'j')
+      .should('contain', 's')
+    })
+
+    it('filters the list of specs when searching for specs', () => {
+      cy.location().should((location) => {
+        expect(location.hash).to.contain('dom-content.spec')
+      })
+
+      cy.get('input').type('dom', { force: true })
+
+      cy.get('[data-testid="spec-file-item"]').should('have.length', 1)
+      .should('contain', 'dom-content.spec')
+
+      cy.get('input').clear().type('asdf', { force: true })
```

Comment:
can we validate the clear functionally worked correctly?

```suggestion
      cy.get('input').clear()
      
      cy.get('[data-testid="spec-file-item"]').should('have.length', ) // how many?

      cy.get('input').type('asdf', { force: true })
```

Reply:
<img width="821" alt="Screenshot 2022-02-03 at 5 52 37 PM" src="https://user-images.githubusercontent.com/27690333/152442666-681f40cb-b78b-4a8b-b190-5edac8971927.png">

I used `force: true` since that's what was done in the component tests for the reporter. Looks like removing it throws an error 😕

Reply:
Hmmm in my experience if the element isn't clickable because it's covered by another element, it's likely a problem with the clickabling for the end-user. Is this input actually interactable at this point in the workflow when manually verifying?

Reply:
That's what's strange- the input interacts perfect fine when I try and type manually at this point in the test. doesn't seem to be covered by anything